officers/harcourt_fenton_mudd_harry.json

```diff
@@ -85,5 +85,89 @@
     "command": 0.03,
     "engineering": 0.05,
     "science": 0.05
-  }
+  },
+  "traits": [
+    {
+      "name": "Escape_Artist",
```

Comment:
Could you remove the underscore here?

Reply:
Will do. I'll also fix my parser to translate this permanently.